### PR TITLE
Family Security Act 2.0 provisions

### LINKED ISSUE #4977: Family Security Act 2.0 provisions
To add:
CTC child cap (6)
CTC phase-in (10k)

EITC max amount increased and limited the increase by number of children 
Removed marriage penalty regarding the phase-out and max credit structure

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #4977

### CODE REVIEW

**Review by MaxGhenis — APPROVED** [review]

File: policyengine_us/reforms/congress/romney/family_security_act_two_point_o/family_security_act_two_point_o.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            p = parameters(
+                period
+            ).gov.contrib.congress.romney.family_security_act_two_point_o.eitc.amount
+            filing_status = tax_unit("filing_status", period)
+            joint = filing_status == filing_status.possible_values.JOINT
+            return where(
+                joint, p.joint.calc(child_count), p.single.calc(child_count)
+            )
+
+    class reform(Reform):
```
Comment (MaxGhenis) [inline policyengine_us/reforms/congress/romney/family_security_act_two_point_o/family_security_act_two_point_o.py:74]: don't we also need to change the list of refundable/non-refundable credits? ctc becomes fully refundable right?

File: policyengine_us/parameters/gov/contrib/congress/romney/family_security_act_two_point_o/ctc/child_cap.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+description: Under the Family Security Act 2.0, the Child Tax Credit will be limited to this number of children per filing unit.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/contrib/congress/romney/family_security_act_two_point_o/ctc/child_cap.yaml:1]: How about this kind of phrasing?
```suggestion
description: Senator Mitt Romney proposed limiting the Child Tax Credit to this number of children per filer under the Family Security Act 2.0.
```

File: policyengine_us/tests/policy/contrib/congress/romney/family_security_act_two_point_o/family_security_act_two_point_o.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+- name: AGI over phase_in amount and joint eitc filers
```
Comment (MaxGhenis) [inline policyengine_us/tests/policy/contrib/congress/romney/family_security_act_two_point_o/family_security_act_two_point_o.yaml:1]:
```suggestion
- name: AGI over phase_in amount and joint EITC filers 
```

File: policyengine_us/parameters/gov/contrib/congress/romney/family_security_act_two_point_o/ctc/phase_in/earnings_limit.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+description: Under the Family Security Act 2.0, the Child Tax Credit will phase-in linearly until this earnings level.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/contrib/congress/romney/family_security_act_two_point_o/ctc/phase_in/earnings_limit.yaml:1]: Also I think `phase-in` is a noun, `phase in` would be the verb
```suggestion
description: Senator Mitt Romney proposed phasing in the Child Tax Credit linearly from zero to this earnings level under the Family Security Act 2.0.
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

File: policyengine_us/reforms/congress/romney/family_security_act_two_point_o/family_security_act_two_point_o.py (hunk 1 above)
Comment (PavelMakarchuk) [reply policyengine_us/reforms/congress/romney/family_security_act_two_point_o/family_security_act_two_point_o.py:74]: I thought we had a switch in the app as the [fully_refundable.yaml](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/irs/credits/ctc/refundable/fully_refundable.yaml)

**Comment by MaxGhenis** [comment]
needs the merge conflict fixed

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**